rest/src/main/java/at/jku/isse/ecco/rest/classes/RestFeature.java

```diff
@@ -0,0 +1,35 @@
+package at.jku.isse.ecco.rest.classes;
+
+import at.jku.isse.ecco.feature.Feature;
+import at.jku.isse.ecco.feature.FeatureRevision;
+
+import java.util.Collection;
+import java.util.LinkedList;
+
+public class RestFeature {
```

Comment:
why do the core classes need json annotations if purpose-made classes already exist?